Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contest-results/ContestResultsPage.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import { useMemo } from 'react';
+import { DataGrid, GridColDef, GridRenderCellParams, GridValueGetterParams } from '@mui/x-data-grid';
+import { useParams } from 'react-router';
+import { useEffect } from 'react';
+import { setLayout } from '../shared/set-layout';
+import { makePrivate } from '../shared/make-private';
+import { useCurrentContestResults } from '../../hooks/contests/use-current-contest-results';
+import { CONTEST_PARTICIPATION_TYPES, CONTEST_RESULT_TYPES } from '../../common/constants'
+import Heading from '../../components/guidelines/headings/Heading';
+import { IContestResultsParticipationProblemType, IContestResultsType } from '../../hooks/contests/types';
+import Hyperlink from '../../components/guidelines/buttons/Hyperlink';
+import _ from 'lodash';

```

Comment:
I link to sort the imports as follows:
```
React
Libs
Guidelines
Types
Components
Styles

Reply:
Can't we just ignore the order like we do in the back end?